### DISCUSSION TITLE
next of delayed startup superweapon

### DISCUSSION THREAD

**Discussion by Fi-Cap** [Phobos-developers/Phobos#1311] [Ideas and feedback]
### Description

I think SW.next could add delayed startup next superweapon.

### INI code

```ini
[SomeSW]
SW.next.delay=0
```

### Checklist

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [x] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this topic.
- [X] I used the GitHub search and read the topic list to find a similar topic and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

**Comment by deathreaperz** [comment]
IMO you can use SW.Deferment for the delay of the next SW.
